app/cdap/components/PipelineDiff/DiffIcon.tsx

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I think we would prefer DiffIcon/index.tsx to DiffIcon.tsx, but I'm ok with this. @seanfeld wdyt

Reply:
I think we voted on this. If I remember correctly, if this component is only being used inside pipelineDiff/index.tsx or its only used locally then I think we're fine with the components like this. @njbriggs do you remember?

Reply:
I think we discussed whether we always require a directory directly under `components`? For this question, I'm with Sean, and this is fine as long as usage is entirely within the component.